src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.UnitConverter.UnitTest/UnitHandlerTests.cs

```diff
@@ -26,6 +26,14 @@ public void HandleLength()
             Assert.AreEqual(100, result);
         }
 
+        [TestMethod]
+        public void HandleSmall()
```

Comment:
"small" what?

Reply:
Renamed to `HandleNanometerToKilometer` the answer is 10⁻¹² and I think it's a small number.

Reply:
Perhaps rename it to `HandleSmallNumber` ;)

Reply:
Nope, converting nanometer to parsec is even smaller. 😼